mean-module/index.js

```diff
@@ -0,0 +1,160 @@
+      message: 'Which client-side folders would you like your module to include?',
+      choices: [{
+        value: 'addConfigFolder',
+        name: 'config',
+        checked: true
+      }, {
+        value: 'addControllersFolder',
+        name: 'controllers',
+        checked: true
+      }, {
+        value: 'addCSSFolder',
+        name: 'css',
+        checked: false
+      }, {
+        value: 'addDirectivesFolder',
+        name: 'directives',
+        checked: false
+      }, {
+        value: 'addFiltersFolder',
+        name: 'filters',
+        checked: false
+      }, {
+        value: 'addImagesFolder',
+        name: 'img',
+        checked: false
+      }, {
+        value: 'addServicesFolder',
+        name: 'services',
+        checked: true
+      }, {
+        value: 'addTestsFolder',
+        name: 'tests',
+        checked: true
+      }, {
+        value: 'addViewsFolder',
+        name: 'views',
+        checked: true
+      }]
+    }, {
+      type: 'checkbox',
+      name: 'serverFolders',
+      message: 'Which server-side folders would you like your module to include?',
+      choices: [{
+        value: 'addServerConfigFolder',
+        name: 'config',
+        checked: false
+      }, {
+        value: 'addServerControllersFolder',
+        name: 'controllers',
+        checked: true
+      }, {
+        value: 'addServerConfigFolder',
+        name: 'models',
+        checked: true
+      }, {
+        value: 'addServerPoliciesFolder',
+        name: 'policies',
+        checked: false
+      }, {
+        value: 'addServerRoutesFolder',
+        name: 'routes',
+        checked: true
+      }, {
+        value: 'addServerTestsFolder',
+        name: 'tests',
+        checked: true
+      }]
+    }];
+
+    this.prompt(prompts, function (props) {
+      // client-side
+      this.addConfigFolder = props.addConfigFolder;
+      this.addControllersFolder = props.addControllersFolder;
+      this.addCSSFolder = props.addCSSFolder;
+      this.addDirectivesFolder = props.addDirectivesFolder;
+      this.addFiltersFolder = props.addFiltersFolder;
+      this.addImagesFolder = props.addImagesFolder;
+      this.addServicesFolder = props.addServicesFolder;
+      this.addTestsFolder = props.addTestsFolder;
+      this.addViewsFolder = props.addViewsFolder;
+
+      //server-side
+      this.addServerConfigFolder = props.addServerConfigFolder;
+      this.addServerControllersFolder = props.addServerControllersFolder;
+      this.addServerConfigFolder = props.addServerConfigFolder;
+      this.addServerPoliciesFolder = props.addServerPoliciesFolder;
+      this.addServerRoutesFolder = props.addServerRoutesFolder;
+      this.addServerTestsFolder = props.addServerTestsFolder;
+
+      done();
+    }.bind(this));
+  },
+
+  renderModule: function () {
+    // Create module folder
+    mkdirp.sync('modules/' + this.slugifiedName + '/client/');
+
+    // Create client module sub-folders
+    if (this.addConfigFolder)
```

Comment:
Added the braces, but I'd say now the code got much less readable and clear because of them.
Also, moved name to prompts here as well, with validation function, checking for empty and invalid (non-string) names.